storage/innobase/include/log0log.h

```diff
@@ -366,9 +377,26 @@ ulint
 log_block_calc_checksum_crc32(
 	const byte*	block);
 
+/** Gets a log block encrypt bit.
+@param[in]	log_block	log block
+@return TRUE if this block was encrypted */
+MY_NODISCARD
+UNIV_INLINE
+bool
+log_block_get_encrypt_bit(
+	const byte*	log_block);
 /** Calculates the checksum for a log block using the "no-op" algorithm.
 @param[in]	block	the redo log block
 @return		the calculated checksum value */
+/** Sets the log block encrypt bit.
+@param[in,out]	log_block	log block
+@param[in]	val		value to set */
+UNIV_INLINE
+void
+log_block_set_encrypt_bit(
```

Comment:
log_block_set_encrypt_bit(byte* log_block, bool val);
Fix only if you have to re-push/re-test otherwise